src/watchdog/src/watchdog.py

```diff
@@ -418,6 +433,9 @@ def main(args):
     parser.add_argument("--interval", "-i", help="interval between two collection", default="30")
     parser.add_argument("--port", "-p", help="port to expose metrics", default="9101")
     parser.add_argument("--hosts", "-m", help="yaml file path contains host info", default="/etc/watchdog/config.yml")
+    parser.add_argument("--tls", "-t", help="access api-server auth or not", default="False")
```

Comment:
Maybe remove tls argument? Since if user call with --ca and --bearer it implies they want to use tls.

Reply:
remove